governance/xc_admin/packages/xc_admin_common/src/contracts/Contract.ts

```diff
@@ -0,0 +1,139 @@
+import { ChainId, Instruction } from "@pythnetwork/xc-governance-sdk";
```

Comment:
Do you see any problem in using `PythGovernanceAction` instead? `Instruction` from `@pythnetwork/xc-governance-sdk`. Goal is enumerating all the available governance actions in one place and using the same class for deserializing them in the proposal page.